File: java/src/main/java/com/hachicore/demo/jooq/actor/ActorFilmography.java
package com.hachicore.demo.jooq.actor;

import com.hachicore.demo.jooq.tables.pojos.Actor;
import com.hachicore.demo.jooq.tables.pojos.Film;
import lombok.Getter;

import java.util.List;

@Getter
public class ActorFilmography {

    private final Actor actor;
    private final List<Film> filmList;

    public ActorFilmography(Actor actor, List<Film> filmList) {
        this.actor = actor;
        this.filmList = filmList;
    }

}

File: kotlin/src/main/kotlin/com/hachicore/demo/jooq/actor/ActorFilmography.kt
package com.hachicore.demo.jooq.actor

import com.hachicore.demo.jooq.tables.pojos.Actor
import com.hachicore.demo.jooq.tables.pojos.Film

data class ActorFilmography(
    val actor: Actor,
    val filmList: List<Film>
)